services/skus/service.go

```diff
@@ -2245,7 +2245,7 @@ func (s *Service) redeemBlindedCred(ctx context.Context, w http.ResponseWriter,
 	case timeLimitedV2:
 		redeemFn = s.cbClient.RedeemCredentialV3
 	default:
-		return handlers.WrapError(fmt.Errorf("credential type %s not suppoted", kind), "unknown credential type %s", http.StatusBadRequest)
```

Comment:
This error and message is returned to clients so to be safe we should leave as is for now. In the past we have had issue with clients relying on the message it self to perform logic.